core/src/main/res/layout/activity_bookmarks.xml

```diff
@@ -7,6 +7,20 @@
 
   <include layout="@layout/layout_standard_app_bar" />
 
+  <TextView
+    android:id="@+id/no_bookmarks"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/no_bookmarks"
+    android:textSize="15sp"
+    android:gravity="center"
```

Comment:
gravity center is doing nothing for a wrap content sized textView positioned in (more or less) the centre of its parent ConstrintLayout. You can delete it